db/migrate/20150911195321_add_budget_roles.rb

```diff
@@ -0,0 +1,24 @@
+class AddBudgetRoles < ActiveRecord::Migration
+  def change
+    User.with_email_role_slug!(
```

Comment:
The best approach may be to have the logic under `lib/` called from both places...the benefit being that you can easily write tests for it.

In past projects, any time required seed data has been added, I've just seen an announcement sent to all of the developers to "make sure to run `rake db:seed`". Having it in a migration is probably better because it's enforced by Rails, but has the downside of being inadvertently broken. See also:

https://twitter.com/aidanfeldman/status/643550323538313216